administrator/components/com_fields/helpers/fields.php

```diff
@@ -170,8 +173,23 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 					{
 						try
 						{
+							/*
+							 *On before field render
+							 *Event allow plugins to modfify the output of the field before it is rendered
+							 */
+                            $dispatcher = JEventDispatcher::getInstance();
+							if($field->value)
+								$dispatcher->trigger('onFieldBeforeRender',array($context,'mustache',&$field));
```

Comment:
Is the form of rendering really needed? The triggers should be independent of the form of rendering. Or do I miss here something?